Sources/SignalProducer.swift

```diff
@@ -77,23 +63,24 @@ public struct SignalProducer<Value, Error: Swift.Error> {
 	/// - parameters:
 	///   - startHandler: A closure that accepts observer and a disposable.
 	public init(_ startHandler: @escaping (Signal<Value, Error>.Observer, Lifetime) -> Void) {
-		self.init { () -> Instance in
+		core = SignalCore {
```

Comment:
It's small, but the other `init`s case `self.init(<core instance>)`.

Can we update this one or the others to make them consistent?